src/main/java/net/mcreator/preferences/data/UISection.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.preferences.data;
+
+import net.mcreator.io.OS;
+import net.mcreator.preferences.PreferencesEntry;
+import net.mcreator.preferences.PreferencesSection;
+import net.mcreator.preferences.entries.BooleanEntry;
+import net.mcreator.preferences.entries.ColorEntry;
+import net.mcreator.preferences.entries.StringEntry;
+import net.mcreator.ui.dialogs.preferences.PreferencesDialog;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.laf.MCreatorTheme;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.EventObject;
+import java.util.List;
+import java.util.Locale;
+import java.util.function.Consumer;
+
+public class UISection extends PreferencesSection {
+
+	public PreferencesEntry<Locale> language;
+	public ColorEntry interfaceAccentColor;
+	public StringEntry backgroundSource;
+	public BooleanEntry aaText;
+	public StringEntry textAntialiasingType;
+	public BooleanEntry usemacOSMenuBar;
+	public BooleanEntry useNativeFileChooser;
+	public BooleanEntry expandSectionsByDefault;
+	public BooleanEntry use2DAcceleration;
+	public BooleanEntry autoReloadTabs;
+	public BooleanEntry remindOfUnsavedChanges;
+	public BooleanEntry discordRichPresenceEnable;
+
+	public UISection(String preferencesIdentifier) {
+		super(preferencesIdentifier);
+
+		interfaceAccentColor = addEntry(new ColorEntry("interfaceAccentColor", MCreatorTheme.MAIN_TINT_DEFAULT));
+		backgroundSource = addEntry(
+				new StringEntry("backgroundSource", "All", "All", "Current theme", "Custom", "None"));
+		aaText = addEntry(new BooleanEntry("aaText", true));
+		textAntialiasingType = addEntry(

```

Comment:
default value of textAntialiasingType can't be All, please recheck others too while you are at it so we don't miss anything

Reply:
I rechecked everything and only another one didn't have the correct default value (`false` instead of `true`)